apps/prairielearn/src/lib/features/index.ts

```diff
@@ -0,0 +1,5 @@
+import { FeatureManager, FeatureGrantType } from './manager';
+
+const features = new FeatureManager(['course:manual-grading']);
```

Comment:
What does the `course:` mean here? Are we going to programmatically enforce that `course:` features can only be enabled for courses (and institutions)? If so, I'm worried about hiding `course:` inside the string where we aren't typechecking it. On the other hand, if we aren't planning to enforce something here, I wonder whether we should bother having the `course:` prefix at all?

Reply:
I was just playing around with naming conventions, I was going to drop this before this was merged anyways. We _had_ talked about programmatically annotating feature flags with a particular scope that they apply to (e.g. we could say rubrics are scoped to "course" and can thus only be applied to "course" or "institution"). I still think that might be a good idea for documentation and runtime safeguards?

Reply:
Talked with @mwest1066 about this offline. We're going to avoid this for now in favor of just being careful. In the future, we can look at adding guardrails.